packages/oats-runtime/package.json

```diff
@@ -23,6 +23,8 @@
   },
   "dependencies": {
     "@smartlyio/safe-navigation": "^5.0.1",
+    "@types/escape-html": "1.0.1",
+    "escape-html": "1.0.3",
```

Comment:
mmm. maybe this is ok as [koa depends](https://www.npmjs.com/browse/depended/escape-html) on it anyway 😁

Reply:
if it's a fixed version, we're quite safe as it's not possible to unpublish a package in npm anymore (unless it has no dependencies and there're only few downloads a week). So no left-pad scenario for us.